From cd1e4344fd39823f17b7c2435e4f93a0936095ea Mon Sep 17 00:00:00 2001
From: Chad Dupuis <cdupuis@redhat.com>
Date: Tue, 19 Aug 2014 19:30:56 -0400
Subject: [scsi] qla4xxx: from treewide: fix comments and printk msgs

Message-id: <1408476658-5056-19-git-send-email-cdupuis@redhat.com>
Patchwork-id: 88347
O-Subject: [PATCH RHEL 7.1 e-stor 18/20] treewide: fix comments and printk msgs
Bugzilla: 1089349
RH-Acked-by: Rob Evers <revers@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Masanari Iida <standby24x7@gmail.com>

scsi commit id 8faaaead62c3c7394fa6302303ce70e484b509ba

This patch fixed several typo in printk from various
part of kernel source.

Signed-off-by: Masanari Iida <standby24x7@gmail.com>
Signed-off-by: Jiri Kosina <jkosina@suse.cz>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/qla4xxx/ql4_83xx.c b/drivers/scsi/qla4xxx/ql4_83xx.c
index 65408e5..556c152 100644
--- a/drivers/scsi/qla4xxx/ql4_83xx.c
+++ b/drivers/scsi/qla4xxx/ql4_83xx.c
@@ -361,7 +361,7 @@ int qla4_83xx_drv_lock(struct scsi_qla_host *ha)
     }
     /* Recovery Failed, some other function
      * has the lock, wait for 2secs and retry */
-    ql4_printk(KERN_INFO, ha, "%s: IDC lock Recovery by %d failed, Retrying timout\n",
+    ql4_printk(KERN_INFO, ha, "%s: IDC lock Recovery by %d failed, Retrying timeout\n",
         __func__, ha->func_num);
     timeout = 0;
    }
-- 
1.7.1